content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+
+A lot of code cleanup and refactoring was done, fixing a few issues and bringing some speed improvements. Among these, the following table, while a little technical, shows the impressive speed improvement between the previous implementation of our hex decoder and the one shipped in Jellyfin 10.5.0.
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+Another example of the speed improvements to the server this cycle is the following comparison between our previous alphanumeric comparator and the one shipped in Jellyfin 10.5.0, showing a reduction of the execution time of more than 50%!
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+**Support AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning and the way base URLs and DLNA are handled. **Raspberry Pi hardware acceleration** for older models is also now supported. Sorry Raspberry Pi 4 owners, this one needs a bit more work to be supported properly.
+
+As part of a project to move the core Metadate providers to plugins, MusicBrainz now allows you to configure the URL of the instance you want to pull data from, thereby allowing you to host an instance of MusicBrainz and sidestep the global rate limiting enforced by the main service.
```

Comment:
```suggestion
As part of a project to move the core Metadata providers to plugins, MusicBrainz is now a default plugin and allows you to configure the URL of the instance you want to pull data from, thereby allowing you to host an instance of MusicBrainz and sidestep the global rate limiting enforced by the main service.
```

Reply:
I'm doing a second pass and caught the Metadata :p I'll change the other bit as well, makes more sense.